packages/core/src/components/popover/popover.tsx

```diff
@@ -499,23 +503,26 @@ export class Popover extends AbstractPureComponent<IPopoverProps, IPopoverState>
         }
     };
 
-    private handleTargetFocus = (e?: React.FormEvent<HTMLElement>) => {
+    private handleTargetFocus = (e?: React.FocusEvent<HTMLElement>) => {
         if (this.props.openOnTargetFocus && this.isHoverInteractionKind()) {
+            if (e.relatedTarget == null && !this.lostFocusOnSamePage) {
+                // ignore this focus event -- the target was already focused but the page itself
+                // lost focus (e.g. due to switching tabs).
+                return;
+            }
             this.handleMouseEnter(e);
         }
     };
 
-    private handleTargetBlur = (e?: React.FormEvent<HTMLElement>) => {
+    private handleTargetBlur = (e?: React.FocusEvent<HTMLElement>) => {
```

Comment:
can we make this arg required? to ensure that it exists before you reference it

Reply:
Sure thing. No idea why it was optional to begin with. API change in React?